### ISSUE TITLE
AccessibilityHint also failing in iOS26

### ISSUE THREAD

**Issue by blackcat-vin** [nalexn/ViewInspector#402]
Hi @andy-vanwagoner-jn Similar to AccessibilityLabel and AccessibilityValue fixes in the PR https://github.com/nalexn/ViewInspector/pull/400 

AccessibilityHint is also failing for iOS26. Please advise if the fix is similar like below i can raise a PR i’m unable to test the change

if #available(ios 26.0, macos 26.0, tvos 26.0, watchos 26.0,*) {
text = try v4AccessibilityPropertyFirst(path: "hint|some|texts", call: call)
}

**Comment by blackcat-vin** [comment]
This has already been handled in 0.10.3 release. Thanks.
